edm4hep.yaml

```diff
@@ -503,6 +503,22 @@ datatypes:
       void setCovMatrix(float value, edm4hep::Cartesian dimI, edm4hep::Cartesian dimJ) { getCovMatrix().setValue(value, dimI, dimJ); }\n
       "
 
+  edm4hep::DriftChamberDigi:
```

Comment:
SensitiveWireHit or DriftChamberHit? 

"DriftChamberDigi" sounds like the algorithm that does the digitisation.

Reply:
Went for SensitiveWireHit